pkg/log/log.go

```diff
@@ -257,6 +258,9 @@ func addRequestInfo(req *http.Request) []interface{} {
 		}
 		fields = append(fields, "req_payload")
 		fields = append(fields, newStr)
+		// Once req.Body is read, it is empty. Restore its contents by
+		// assigning a new reader with the same contents.
```

Comment:
🙌 for putting on comment here, it's good to remember why we have this in the future 😅